src/components/App/App.js

```diff
@@ -1,9 +1,22 @@
+import { useState } from 'react'
 import styled from 'styled-components'
+import { serviceData as Data } from '../../serviceData.json'
```

Comment:
By write `Date` with a capital letter, one might think it's a component instead of a variable holding data. Also, you could consider using the name `serviceDate` as it is more descriptive. But that is only my opinion :)
```suggestion
import { serviceData as data } from '../../serviceData.json'
```